app/controllers/person_relations_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
People::RelationsshipsController ?
People::RelationsBaseController ?

PersonRelationsBaseController ?

maybe the name Base would indicate better that this is an abstract controller ?!

put it in People namespace?

just some ideas/inputs ... we might also discuss it

Reply:
i took the name from the old controller "Api::PersonRelationsController"